src/constants.js

```diff
@@ -9,15 +9,20 @@ const GITHUB_REPOSITORY = process.env["GITHUB_REPOSITORY"];
 
 const REPO_DIRECTORY = process.env["GITHUB_WORKSPACE"];
 
+const JPEG_QUALITY = process.env["INPUT_JPEGQUALITY"] ? process.env["INPUT_JPEGQUALITY"] : 80;
+const PNG_QUALITY = process.env["INPUT_PNGQUALITY"] ? process.env["INPUT_PNGQUALITY"] : 80;
+const WEBP_QUALITY = process.env["INPUT_WEBPQUALITY"] ? process.env["INPUT_WEBPQUALITY"] : 80;
+const IGNORE_PATHS = process.env["INPUT_IGNOREPATHS"] ? process.env["INPUT_IGNOREPATHS"].split(",") : ["nodemodules/**"];
+
 const COMMITTER = {
   name: "Calibre",
   email: "hello@calibreapp.com"
 };
 
 if (!REPO_DIRECTORY) {
-  console.log("There is no GITHUB_WORKSPACE environment variable");
+  console.log("::error::There is no GITHUB_WORKSPACE environment variable");
   process.exit(1);
-}
+};
 
 const CONFIG_PATH = path.join(
```

Comment:
Should `CONFIG_PATH` be removed from here too?

Reply:
nope. it's still needed otherwise we break exsiting functionality.